test/system/profile_test.rb

```diff
@@ -103,4 +103,17 @@ def setup
     assert_text I18n.t(:'profiles.personal_access_tokens.index.active_personal_access_tokens',
                        count: @active_token_count - 1)
   end
+
+  test 'can view language selection' do
+    visit profile_preferences_path
+
+    assert_text I18n.t(:'profiles.preferences.locale_form.en')
+  end
+  test 'can update language to french' do
+    visit profile_preferences_path
+
+    find('input[id=user_locale_fr]').click
+
+    assert_text 'Anglais'
```

Comment:
I missed one thing sorry, can you check that the flash alert displays when the locale is updated, and check for the success text within it

Reply:
Fixed in 40a49b116b2e7f0f55b16806e4208811e3333b02